src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -437,13 +460,122 @@ internal PackageManagerSearchViewModel()
             SelectedHosts = new List<string>();
         }
 
+        /// <summary>
+        /// Initialize Lucene config file writer.
+        /// </summary>
+        private void InitializeLuceneConfig()
```

Comment:
do you think is possible to move this method to LuceneSearchUtility class?
I can see the code is very similar than the one in DynamoModel.cs so it can be re-used in several places.